pkg/controller/expcertificates/internal/secretsmanager/secret_test.go

```diff
@@ -270,24 +271,23 @@ func TestSecretsManager(t *testing.T) {
 
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			fixedClock.SetTime(fixedClockStart)
-			test.builder.Clock = fixedClock
+			test.builder.Clock = internaltest.FixedClock
```

Comment:
Hm, I'm not sure if moving the FixedClock into internaltest is that helpful - it hides what we're actually doing with the clock and isn't exactly complex boilerplate to create. Can we keep it here to keep these tests a bit more obvious as to what's happening? It's not clear that `fixedClock.SetTime(fixedClockStart)` is being called before each test case anymore...

Reply:
Perhaps have `MustCreateCryptoBundle` accept a Clock?